gemrb/plugins/FXOpcodes/FXOpcodes.cpp

```diff
@@ -4690,16 +4690,7 @@ int fx_cast_spell (Scriptable* Owner, Actor* target, Effect* fx)
 	}
 
 	if (fx->Parameter2 == 0 || target->Type == ST_CONTAINER) {
-		// no deplete, no interrupt, caster or provided level
-		// ForceSpell doesn't have a RES variant, so there's more work
-		std::string tmp = fmt::format("ForceSpell([-1],{})", ResolveSpellNumber(fx->Resource));
-		Action *forceSpellAction = GenerateActionDirect(std::move(tmp), target);
-		if (fx->Parameter1 != 0) {
-			// override casting level
-			forceSpellAction->int1Parameter = fx->Parameter1;
-		}
-		Owner->AddActionInFront(forceSpellAction);
-		Owner->ImmediateEvent();
```

Comment:
What was the problem? This effect deals with a lot more use cases ...

Reply:
> What was the problem? This effect deals with a lot more use cases ...

Look at the cleric spell "find traps". This is an effect with delayed sub-effects (20 pieces). For each sub-effect, the character stopped and did a full cast animation.

Reply:
Hmm, ok, needs a closer look then, thanks.

Reply:
The casting problem has been gone, but the spell payload itself indeed does not work. Probably a projectile thing, still investigating.
This code change could not have fixed it though; the spell uses Parameter2 == 1 and self-targeting, so this branch doesn't get used.